pkg/analysis/passes/brokenlinks/brokenlinks.go

```diff
@@ -149,6 +149,9 @@ func run(pass *analysis.Pass) (interface{}, error) {
 			defer resp.Body.Close()
 
 			if resp.StatusCode != http.StatusOK {
+				if isGitHubURL(url.url) && strings.Contains(resp.Status, "429 Too Many Requests") {
```

Comment:
Maybe just have a check for status code on its own here since we probably don't care about any 429s in general do we?

```suggestion
				if resp.StatusCode == http.StatusTooManyRequests {
```